provider/blockdev_snapshot_base.py

```diff
@@ -53,6 +53,10 @@ def prepare_snapshot_file(self):
             if self.params.get("mode") == "existing":
                 self.snapshot_image.create()
 
+    def data_disk_attached(self):
```

Comment:
You can call  params.objects("images") to get image list directly.
return  len(self.main_vm.params.objects("images")) > 1

Reply:
have updated.